src/index.ts

```diff
@@ -80,6 +83,28 @@ function formatInstallResults(skipFailures): string {
     .join(', ');
 }
 
+function formatDependencyInfoAnalysis(): string {
+  let output = '';
+  const {direct, shared} = dependencyInfo;
```

Comment:
In a future PR, I'd love to move this variable out of the global context and pass it in as a function argument instead. But, since we have a few other global variables like this one already, this is totally fine to leave as-is for now.

Reply:
Agreed, I only made this global to copy the currently existing pattern. I'm happy to move this into an argument now, however, it should be pretty easy to do so?